worlds/witness/locations.py

```diff
@@ -0,0 +1,281 @@
+"""
+Defines constants for different types of locations in the game
+"""
+
+
+from worlds.witness.Options import is_option_enabled
+from worlds.witness.full_logic import ParsedWitnessLogic
+
+
+class WitnessLocations():
+    """Class that handles Witness Locations"""
```

Comment:
purely stylistic, since all other docstrings in here look like this
```suggestion
    """
    Class that handles Witness Locations
    """
```